src/database/sql_database.cc

```diff
@@ -1797,7 +1797,7 @@ void SQLDatabase::addAutoscanDirectory(std::shared_ptr<AutoscanDirectory> adir)
       << quote(AutoscanDirectory::mapScanmode(adir->getScanMode())) << ','
       << mapBool(adir->getRecursive()) << ','
       << mapBool(adir->getHidden()) << ','
-      << quote(adir->getInterval()) << ','
+      << quote(adir->getInterval().count()) << ','
```

Comment:
count is not guaranteed to be seconds right?, its just the count of the "Interval"

Reply:
I believe it is. getInterval() returns std::chrono::seconds. My reading of https://en.cppreference.com/w/cpp/chrono/duration/count indicates count() just converts it to an integer.